deployment/nginx.coffee

```diff
@@ -365,6 +365,17 @@ module.exports.create = (KONFIG, environment)->
         resolver 8.8.8.8;
       }
 
+      # redirect /s3/:bucket/:key to :bucket.s3.amazonaws.com/:key
+      location ~^/s3/([^/]*)/(.*)$ {
```

Comment:
thanks for the very generalized solution but this might give opportunities to malicious users eg they can abuse our bandwidth with big files  

What do you think of using smth like `~*(^\/s3\/koding-(klient|softlayer|provision))/(.*)$` to make it arguably less prone?

ps: didnt test the regex 😇